.teamcity/DFastBETests/buildwithcommandwindow.kt

```diff
@@ -0,0 +1,73 @@
+import jetbrains.buildServer.configs.kotlin.*
+import jetbrains.buildServer.configs.kotlin.buildSteps.script
+import jetbrains.buildServer.configs.kotlin.failureConditions.BuildFailureOnMetric
+import jetbrains.buildServer.configs.kotlin.failureConditions.BuildFailureOnText
+import jetbrains.buildServer.configs.kotlin.failureConditions.failOnMetricChange
+import jetbrains.buildServer.configs.kotlin.failureConditions.failOnText
+import DFastCleanConfiguration
+
+object BuildWithCommandWindow : BuildType({
+    templates(DFastCleanConfiguration)
+    id("BuildWithCommandWindow")
+    name = "Build with command window"
+
+    artifactRules = """
+        dfastbe.dist => dfastbe.zip
+        compilation-report.xml => .
+    """.trimIndent()
+
+    vcs {
+        root(DslContext.settingsRoot)
+    }
+
+    steps {
+        script {
+            name = "build D-FAST BE"
+            id = "build_D_FAST_BE"
+            scriptContent = """
+                CALL conda activate %CONDA_ENV_NAME%
+                CALL .\BuildScripts\BuildDfastbe.bat
```

Comment:
can you add || exit/b (i.e. `CALL .\BuildScripts\BuildDfastbe.bat || exit /b`) to make it fail if the command reurns non-zero exit code ?